designs/2022-community-eslint-org/README.md

```diff
@@ -133,53 +133,71 @@ I personally think that the acceptance criteria for a package to go under the
   [`eslint-plugin-flowtype`](https://github.com/gajus/eslint-plugin-flowtype)
   and/or [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc)
   if they would feel it would be better maintained by the community instead of
-  putting all workload on their shoulders.
+  putting all workload on their shoulders. I can see the same happening for
+  [`eslint-plugin-security`](https://github.com/nodesecurity/eslint-plugin-security),
+  which is currently maintained by the
+  [@nodesecurity](https://github.com/nodesecurity) team.
+
+- It should agree to the code of conduct of the org
+- It should have an OSI-approved license
+- `eslintbot` should be added as an owner to the original npm package
 
 If a package is meeting these criteria, but we can't get into contact with the
 owner of the repo to transfer ownership over to the new org, we could
-temporarily fork the repo & publish the package as
-`@eslint-community-fork/<package name>` or so.
+temporarily fork the repo into the new org. However, the ultimate goal would
+always be to transfer the original repo to the new org.  
+If we had a temporary fork, we should create PRs to the original repo to bring
+it up to date with the fork before continuing with other tasks.
 
 ### What permissions should maintainers have?
 
-I think it would be best to have some sort of a "core" team (of at least 3(?)
-people) that's making the decisions in terms of acceptance to the repo, which
-would (in addition to ESLint TSC) all be admin of the org.
+I think it would be best to have some sort of a community core team (of at least
+3(?) people) that's making the decisions in terms of acceptance to the repo,
+which would (in addition to ESLint TSC) all be admins of the org.
+
+We can make a team (with sub-teams, if we want) for each package, where we give
+maintainer rights on the repo for that specific package, but keep admin rights
+to ESLint TSC & the community core team. This way we prevent that the repo would
+be destroyed from the org.
+
+We'll create a setup where we publish new versions of the package with
+`eslintbot`. That way we can keep publishing new versions if someone would
+dissapear.
 
-We can make a team (with sub-teams if we want) for each package, where we give
-admin rights on the repo for that specific package.
+The community core team would help maintain all repos, whereas people in
+specific teams would only help maintain these repositories.
 
-The "core" team would help maintain all repos, whereas people in specific teams
-would only help maintain these repositories.
+The community core team would be responsible of administer the Code of Conduct
+of the organization.
 
-The "core" team would also be responsible to contact package maintainers
+The community core team would also be responsible to contact package maintainers
```

Comment:
Let’s be very careful around this. I don’t think we want to be in the business of pulling packages from people. I’d like to see some more stringent criteria here, like maybe if a package hasn’t had a release in six months then we give the maintainers 30 days to respond to an email.